backend/src/telegram/models/telegram-subscriber.ts

```diff
@@ -8,9 +8,15 @@ import { Column, Model, Table, DataType } from 'sequelize-typescript'
 export class TelegramSubscriber extends Model<TelegramSubscriber> {
```

Comment:
```suggestion
export class TelegramSubscriber extends Model<TelegramSubscriber> {
  @HasMany(() => BotSubscriber, { as: 'bot_subscriber' })
```

I added this association also while implementing the backend. Sorry, forgot to create a PR at that point.